back/src/infra/utils/format-month.ts

```diff
@@ -0,0 +1,20 @@
+export class FormatMonth {
+  private static monthNames = [
+    'Janeiro',
+    'Fevereiro',
+    'Março',
+    'Abril',
+    'Maio',
+    'Junho',
+    'Julho',
+    'Agosto',
+    'Setembro',
+    'Outubro',
+    'Novembro',
+    'Dezembro',
+  ];
```

Comment:
monthNames is already defined in `back/src/application/dashboard/view-model/dashboard-view-model.ts`